tilauskasittely/rahtikirja_pdf.inc

```diff
@@ -353,7 +353,7 @@ for ($tulostuskpl=1; $tulostuskpl<=$tulostakolli; $tulostuskpl++) {
     $pdf->draw_rectangle(711, 60, 689, 300,                      $firstpage, $rectparam);
     $pdf->draw_text(65,  692,  $rakir_row["osoite"],                $firstpage, $kirj);
     $pdf->draw_rectangle(689, 60, 667, 300,                      $firstpage, $rectparam);
-    $pdf->draw_text(65,  670,  $rakir_row["postino"]."  ".$rakir_row["postitp"]."  ".$rakir_row["maa"]."    ".$rakir_row["toim_puh"],  $firstpage, $kirj);
+    $pdf->draw_text(65,  670,  $rakir_row["postino"]."  ".$rakir_row["postitp"]."  ".$rakir_row["maa"]."    ".$postirow["toim_puh"],  $firstpage, $kirj);
```

Comment:
@tonyselkamo Onko oikein, että toim_puh haetaan eri muuttujasta kuin muut toimitusositteen tiedot? Jos ok, niin miksi näin?